src/pages/Login.js

```diff
@@ -16,6 +16,7 @@ const Login = ({ afterLogin }) => {
     if (responseSucceeded) {
       setEmail('')
       setPassword('')
+      // This will call the setUserNameAfterLogin function in the HeaderContnent component to display the Welcome message in the header
       afterLogin(loggedInUser)
```

Comment:
Here we're calling the `afterLogin` function that was passed via props to this component.

In HeaderContent.js , the actual name of the function we passed into this variable is `setUserNameAfterLogin`. That function will receive the logged in user object, and use that to set the welcome message